docs/cluster-management/uninstall.md

```diff
@@ -42,3 +42,7 @@ aws s3 rb --force s3://<bucket>
 # delete the log group (replace <log_group> with what was configured during installation, default: cortex)
 aws logs describe-log-groups --log-group-name-prefix=<log_group> --query logGroups[*].[logGroupName] --output text | xargs -I {} aws logs delete-log-group --log-group-name {}
 ```
+
+If you've setup API gateway and want to delete it, please follow these cleanup [instructions](../guides/api-gateway.md#cleanup).
```

Comment:
nit: I believe `setup` should be `set up`, since I think `setup` is a noun and `set up` is a verb (also below)